_layouts/post.html

```diff
@@ -1,22 +1,8 @@
 ---
-layout:    default
-isArticle: true
-fixedHeader: true
+layout: default
 ---
-
-{% assign minutes = content | number_of_words | divided_by: 120 %}
```

Comment:
This was superfluous in this layout; it wasn’t needed since [ea9b316](https://github.com/skybet/skybet.github.io/commit/ea9b3166e79076ba7f8c62a1c3a01fff2ed649e2#diff-663f387b6a1a407ab38de055a12bc7c8).